tests/Database/SelectFields/ComputedPropertiesTests/UsersQuery.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Database\SelectFields\ComputedPropertiesTests;
+
+use Closure;
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Query;
+use GraphQL\Type\Definition\ResolveInfo;
+use Rebing\GraphQL\Support\SelectFields;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Tests\Support\Models\User;
+
+class UsersQuery extends Query
+{
+    protected $attributes = [
+        'name' => 'users',
+    ];
+
+    public function type(): Type
+    {
+        return Type::listOf(GraphQL::type('User'));
+    }
+
+    public function resolve($root, $args, $contxt, ResolveInfo $info, Closure $getSelectFields)
+    {
+        /** @var SelectFields $selectFields */
+        $selectFields = $getSelectFields();
+
+        return User
+            ::query()
```

Comment:
I think this can just start with `::select(…)`, not requiring the `::query()` at all

Reply:
👍